impl/src/planning/subplanners.rs

```diff
@@ -155,7 +128,7 @@ impl<'planner> WorkspaceSubplanner<'planner> {
     &self,
     node: &Node,
     catalog: &CrateCatalog,
-  ) -> Option<Result<CrateContext>> {
+  ) -> Option<Result<(CrateContext, bool)>> {
```

Comment:
Is this not information that already lives (or could already live) in `CrateContext`? Admittedly that struct could use some cleaning but whether or not the crate is a workspace member seems like it makes sense to store in the context.
https://github.com/google/cargo-raze/blob/ee2effee5b6c95cfcfd5e496677ce61dcd35c472/impl/src/context.rs#L116-L153